stl/inc/xatomic.h

```diff
@@ -19,6 +19,9 @@ _STL_DISABLE_CLANG_WARNINGS
 #pragma push_macro("new")
 #undef new
 
+#define _CONCATX(x, y) x##y
+#define _CONCAT(x, y)  _CONCATX(x, y)

```

Comment:
$20 says we break something in RWC by moving `_CONCAT` =) [No change requested.]